src/SimpleCalculator/UserInputs.cs

```diff
@@ -1,17 +1,23 @@
 namespace SimpleCalculator;
 
 public class UserInputs
-
 {
-    public static int GetUserInput(string message)
+    private string userInput;
+    public string UserInput
+    {
+        get { return userInput; }
+        set { userInput = value; }
+    }
+    
+    public int GetUserInput(string userInstruction)
     {
-        System.Console.WriteLine(message);
-        string? input = Console.ReadLine();
+        System.Console.WriteLine(userInstruction);
```

Comment:
too many responsibilities for one method: it reads console input and validates it. Please refactor it.